### ISSUE TITLE
ChefModernize/DefaultActionFromInitialize false positive

### ISSUE THREAD

**Issue by tas50** [chef/cookstyle#704]
False positive with this in a library:

```ruby
  def log_vars(action, status)
    @type = 'cpe_nomad'
    @action = action
    @status = status
  end
```
